src/main/assembly/dist/cfg/application.properties

```diff
@@ -4,7 +4,7 @@ deposits.permissions=rwxrwx---
 tempdir=/var/opt/dans.knaw.nl/tmp/easy-sword2
 # Require at least some 2 G  spare disk space
 tempdir.margin-available-diskspace-mb=2000
-base-url=http://yourdomain-here.org/
+base-url=http://test.dans.knaw.nl/
```

Comment:
I disagree. When creating "reasonable defaults" picture the scenario of someone installing the rpm via a `yum install`, but you don't know anything else about them or their environment. The base-url of their server is therefore something that you really cannot predict.